modules/core/src/main/java/org/apache/ignite/internal/managers/deployment/GridDeploymentCommunication.java

```diff
@@ -337,50 +337,63 @@ void sendUndeployRequest(String rsrcName, Collection<ClusterNode> rmtNodes) thro
                 GridIoPolicy.P2P_POOL);
         }
     }
-
+    
+    /**
+     * Queries whether a node is a direct or indirectly responsible for 
+     * the request being processed by this thread.
+     * 
+     * @param  node The node in question
```

Comment:
Remove @param and @return as they are redundant to the above contract.

Reply:
Looking at their coding  style, it seems that they require @param and @return (but it is not important that they add value to the reader).     I'm willing to add stuff that they may not want, but I'm going to try to stick with their convention by specifying all of these.